packages/atlas/src/views/studio/MyChannelView/tabs/GeneralTab/GeneralTab.tsx

```diff
@@ -0,0 +1,236 @@
+
+const PUBLIC_SELECT_ITEMS: SelectItem<boolean>[] = [
+  { name: 'Public', value: true },
+  { name: 'Unlisted (channel will not appear in feeds and search)', value: false },
+]
+
+export const GeneralTab = ({ actionBarPortal }: { actionBarPortal: RefObject<HTMLDivElement> }) => {
+  const { channelId } = useUser()
+  const { displaySnackbar } = useSnackbar()
+  const smMatch = useMediaMatch('sm')
+  const nodeConnectionStatus = useConnectionStatusStore((state) => state.nodeConnectionStatus)
+
+  const {
+    channel,
+    loading,
+    error,
+    refetch: refetchChannel,
+  } = useFullChannel(
+    channelId || '',
+    {
+      skip: !channelId,
+      onError: (error) =>
+        SentryLogger.error('Failed to fetch channel', 'CreateEditChannelView', error, {
+          channel: { id: channelId },
+        }),
+    },
+    { where: { channel: { isPublic_eq: undefined, isCensored_eq: undefined } } }
+  )
+  const {
+    feeLoading,
+    form,
+    hasCoverUploadFailed,
+    hasAvatarUploadFailed,
+    hideActionBar,
+    fee,
+    actions: { handleDeleteAvatar, handleAvatarChange, handleDeleteCover, handleCoverChange, handleSubmit },
+    refs: { coverDialogRef, avatarDialogRef },
+  } = useChannelForm({ type: 'edit', refetchChannel, channel })
+  const {
+    register,
+    control,
+    reset,
+    formState: { errors, isDirty },
+  } = form
+
+  if (error) {
+    return <ViewErrorFallback />
+  }
+
+  return (
+    <>
+      <form>
+        <EntitySettingTemplate
+          title="Channel branding"
+          description="Show your followers what your channel is about with customized avatar, cover & description."
+        >
+          <InputsWrapper>
+            <FormField label="Channel name">
+              <TextInput inputSize="large" {...register('title')} />
+            </FormField>
+
+            <Controller
+              name="avatar"
+              control={control}
+              render={({ field: { value } }) => (
+                <FormField label="Channel avatar" description="Max file size is 5MB.">
+                  <StyledAvatar
+                    assetUrl={loading ? null : value.croppedUrl || value.originalUrl}
+                    hasAvatarUploadFailed={hasAvatarUploadFailed}
+                    size={smMatch ? 136 : 88}
+                    onClick={() => {
+                      avatarDialogRef.current?.open(
+                        value.originalBlob,
+                        value.imageCropData || undefined,
+                        !!value.originalBlob
+                      )
+                    }}
+                    editable
+                    loading={loading}
+                  />
+                  <ImageCropModal
+                    imageType="avatar"
+                    onConfirm={handleAvatarChange}
+                    onError={() =>
+                      displaySnackbar({
+                        title: 'Cannot load the image. Choose another.',
```

Comment:
```suggestion
                        title: 'Cannot load the image. Please choose another one.',
```
Smth like this unless it wasn't specified in the in mockups.